### DISCUSSION TITLE
找不到postJson方法

### DISCUSSION THREAD

**Discussion by dongzheng911** [w7corp/easywechat#2465] [Q&A]
use EasyWeChat\Pay\Application;

$app = new Application($config);
        $response = $app->getClient()->postJson("v3/pay/transactions/jsapi", [
            "mchid" => "1518700000", // <---- 请修改为您的商户号
            "out_trade_no" => "native12177525012012070352333'.rand(1,1000).'",
            "appid" => "wx6222e9f48a0xxxxx", // <---- 请修改为服务号的 appid
            "description" => "Image形象店-深圳腾大-QQ公仔",
            "notify_url" => "https://weixin.qq.com/",
            "amount" => [
                "total" => 1,
                "currency" => "CNY"
            ],
            "payer" => [
                "openid" => "o4GgauInH_RCEdvrrNGrnxxxxxx" // <---- 请修改为服务号下单用户的 openid
            ]
        ]);

ide提示找不到postjson方法。
client 里面确实没有这个方法。删掉了？文档没更新吗？

**Comment by dongzheng911** [comment]
改成腾讯云源，解决了。。。。。。。

**Reply by fannan1991** [reply]
> 改成腾讯云源，解决了。。。。。。。

果然解决了，好坑啊，浪费一天时间，用阿里云的源方法不存在不说，用post 定义的xml 根本请求不通啊

**Reply by overtrue** [reply]
阿里云的源停止维护了
